app/src/main/java/com/kuit/kuit6android/ui/cart/component/totalamount/TotalResultRow.kt

```
@@ -0,0 +1,48 @@
+) {
+    Row(
+        modifier = modifier.fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Text(
+            text = "결제예정금액",
+            style = CoupangEatsTheme.typography.head_04_SB_14,
+            color = CoupangEatsTheme.colors.black
+        )
+        Text(
+            text = "${String.format("%,d", totalAmount)}원",
```

detekt (1.23.8)

[warning] String.format("%,d", totalAmount) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)